### ISSUE TITLE
Focus rings on inputs/buttons are missing

### ISSUE THREAD

**Issue by raucao** [67P/hyperchannel#286]

**Comment by galfert** [comment]
When testing locally they all seem to be there. Tested in both Chrome and Firefox.

**Comment by raucao** [comment]
Nice! So it seems like it's already fixed but not deployed yet. In order to deploy from master, we need to wait for the Sockethub 2.0 beta to be released...
